plugins/zoom/help.md

```diff
@@ -302,6 +312,7 @@ Example output:
 
 # Version History
 
+* 4.1.6 - Adding better handling to `monitor_sign_in_out_activity` if the users account does not have all of the required permsions 
```

Comment:
Typo in permissions. Update the comment to include task e.g. Added better handling to 'monitor_sign_in_out_activity' task.

Reply:
This version message has been updated